templates/base.html

```diff
@@ -76,6 +76,7 @@
 				</a>
 			</div>
 		</footer>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
in these cases it's better to use protocol relative links (just remove the "https:"), so the same protocol that's used to load the page is used to access the CDN as well (usually, at least)...